.github/workflows/debian_package.yml

```
@@ -95,3 +95,51 @@ jobs:
         with:
           name: ${{env.PKG_NAME}}_${{ env.RELEASE_VERSION }}_${{ matrix.dist }}.deb
           path: ${{matrix.package}}/build/${{env.PKG_NAME}}_${{ env.RELEASE_VERSION }}_${{ matrix.package == 'cveapi' && 'all' || 'amd64' }}.deb
+
+  add-debian-packages-to-release:
+    permissions:
+      contents: write
+    needs: build
+    runs-on: ubuntu-24.04
+    if: startsWith(github.ref, 'refs/tags/')
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Generate version.py and set RELEASE_VERSION
+        run: |
+          pip install setuptools-scm==7.1.0
+          echo "RELEASE_VERSION=$(python -m setuptools_scm | sed s/rc/~rc/)" >> $GITHUB_ENV
+
+      - name: Download all assets for debian12
+        uses: actions/download-artifact@v4
+        with:
+          path: kat-debian12-${{ env.RELEASE_VERSION }}
+          merge-multiple: true
+          pattern: kat-*_debian12.deb
+
+      - name: Download all assets for ubuntu22.04
+        uses: actions/download-artifact@v4
+        with:
+          path: kat-ubuntu22.04-${{ env.RELEASE_VERSION }}
+          merge-multiple: true
+          pattern: kat-*_ubuntu22.04.deb
+
+      - name: Download cve api asset
+        uses: actions/download-artifact@v4
+        with:
+          name: kat-cveapi_${{ env.RELEASE_VERSION }}_debian12.deb
+
+      - name: Bundle assets
+        run: |
+          tar -cvzf kat-debian12-${{ env.RELEASE_VERSION }}.tar.gz kat-debian12-${{ env.RELEASE_VERSION }}
+          tar -cvzf kat-ubuntu22.04-${{ env.RELEASE_VERSION }}.tar.gz kat-ubuntu22.04-${{ env.RELEASE_VERSION }}
```

This will create a tarball without a directory which what we currently have:

```suggestion
          tar -cvzf kat-debian12-${{ env.RELEASE_VERSION }}.tar.gz -C kat-debian12-${{ env.RELEASE_VERSION }} .
          tar -cvzf kat-ubuntu22.04-${{ env.RELEASE_VERSION }}.tar.gz -C kat-ubuntu22.04-${{ env.RELEASE_VERSION }} .
```